<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Radio">
     <attribute name="title">
      <string>Radio</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Campo_entrada_1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>OPCIÃ“N SELECCIONADA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Check">
     <attribute name="title">
      <string>Check</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Gato</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Campo_entrada_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Slider">
     <attribute name="title">
      <string>Slider</string>
     </attribute>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
     </widget>
     <widget class="QScrollBar" name="hScrollBar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSum">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Suma</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>104</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
